src/sentry/models/group.py

```diff
@@ -419,6 +425,16 @@ class Group(Model):
         ),
         db_index=True,
     )
+    substatus = BoundedPositiveIntegerField(
```

Comment:
Let's not use `PositiveInteger`. It'll apply a check constraint to the whole column and I've seen it lock the table while adding it on large tables before.